Makefile

```
@@ -125,8 +125,8 @@ e2e_ios: $(TEST_IOS_APP)
 
 .PHONY: e2e_ios_local
 e2e_ios_local: $(TEST_IOS_APP)
-	ideviceinstaller --uninstall com.bugsnag.TestFixture
-	bundle exec maze-runner --app=$< --farm=local --os=ios --os-version=14 --apple-team-id=372ZUL2ZB7 --udid="$(shell idevice_id -l)" $(TESTS) --color
+	ideviceinstaller --uninstall om.bugsnag.examples.UnrealTestFixture
```

Looks like an accidental delete to me
```suggestion
	ideviceinstaller --uninstall com.bugsnag.examples.UnrealTestFixture
```

---

Oh weird, maybe not?